src/gatsby-ssr.js

```diff
@@ -1,12 +1,10 @@
-import path from "path";
-
 import { ChunkExtractor } from "@loadable/server"
 
-import { LOADABLE_STATS_FILE_PATH } from "./constant"
+import stats from "/.cache/page-ssr/loadable-stats-build-javascript.json"
```

Comment:
Starting with `/` means from root of project. We can't import constants here tho, else webpack will have trouble figuring out what to bundle - hence hardcoding that path for webpack